src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -199,24 +199,36 @@
       <OtherRunnerScriptArgs>--perf-runner Microsoft.DotNet.xunit.performance.runner.Windows $(OtherRunnerScriptArgs)</OtherRunnerScriptArgs>
     </PropertyGroup>
     <!-- now gather the perf tests -->
+    <!-- All built tests are under bin/tests dir -->
+    <PropertyGroup Condition="'$(IsTestsBinDirPath)'=='true'">
```

Comment:
Any name that is a totally new property that is this confusing needs to be explained nearby in comments.